Revising Send and Sync bounds was a breaking change

`orx-parallel` won't compile anymore `orx-concurrent-bag` version 2.14.0.

It compiles fine with version 2.13.0.

I get errors like this:

```
error[E0277]: `<Vo as values::Values>::Item` cannot be sent between threads safely
   --> /home/z/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/orx-parallel-2.4.0/src/runner/thread_runner_compute/collect_arbitrary.rs:82:29
    |
82  |                         bag.push(x);
    |                             ^^^^ `<Vo as values::Values>::Item` cannot be sent between threads safely
    |
    = help: the trait `Send` is not implemented for `<Vo as values::Values>::Item`
note: required by a bound in `ConcurrentBag::<T, P>::push`
   --> /home/z/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/orx-concurrent-bag-2.14.0/src/bag.rs:542:12
    |
540 |     pub fn push(&self, value: T) -> usize
    |            ---- required by a bound in this associated function
541 |     where
542 |         T: Send,
    |            ^^^^ required by this bound in `ConcurrentBag::<T, P>::push`
help: consider further restricting the associated type
    |
66  |     P: IntoConcurrentPinnedVec<Vo::Item>, <Vo as values::Values>::Item: Send
    |                                           ++++++++++++++++++++++++++++++++++
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "orx-concurrent-bag"
-version = "2.14.0"
+version = "3.0.0"
 edition = "2024"
 authors = ["orxfun <orx.ugur.arikan@gmail.com>"]
 description = "An efficient, convenient and lightweight grow-only concurrent data structure allowing high performance concurrent collection."

